packages/palette-tokens/README.md

```diff
@@ -0,0 +1,46 @@
+# `palette-tokens`
+
+The [design tokens](https://www.lightningdesignsystem.com/design-tokens/) that power [Artsy](https://www.artsy.net)'s [Palette](https://palette.artsy.net).
+
+This package isn't intended to be consumed directly, but rather included as a transitive dependency of palette.
+
+## Usage
+
+In the main theme file...
+
+```
+import buildTheme from "@artsy/palette-tokens"
+export * from "@artsy/palette-tokens";
+import { fontFamily } from "./platform/fonts"
+
+export const themeConfig = buildTheme(fontFamily);
```

Comment:
The undocumented requirement (`import { fontFamily } from "./platform/fonts"`) is a bit confusing. Can we elaborate on what this is, and out of curiosity what would this look like in eigen?

Reply:
The `fontFamily` object in [eigen](https://github.com/artsy/eigen/blob/master/src/palette/platform/fonts/fontFamily.ts) and [palette](https://github.com/artsy/palette/blob/master/packages/palette/src/platform/fonts/fontFamily.ts) vary in definition and shape. Instead of trying to account for the shape differences, I opted to let the platforms deal w/ it. 

That said, API your proposing is fine too 👍